src/lib/lightning/clightning/clightningService.ts

```diff
@@ -107,8 +99,8 @@ export class CLightningService implements LightningService {
         const status = ChannelStateToStatus[c.state];
         return {
           pending: status !== 'Open',
-          uniqueId: c.shortChannelId || c.channelId,
-          channelPoint: `${c.fundingTxid}:${c.fundingOutnum}`,
+          uniqueId: c.channelId.slice(-12),
+          channelPoint: c.channelId,
```

Comment:
This makes me a bit uncomfortable to store the channelId in the channelPoint field. I'm not sure right now how to implement this better. I'll put some thought into it and respond later.

Reply:
The same is status quo for [eclair](https://github.com/jamaljsr/polar/blob/master/src/lib/lightning/eclair/eclairService.ts#L103). 
Another option will be to add a new channelId field, however, this will trickle down into a lot of files and we would need to check the node implementation in order to know which field to access for the UI, i.e channelPoint or channelId.